drivers/display/display_otm8009a.c

```diff
@@ -185,8 +185,7 @@ static int otm8009a_configure(const struct device *dev)
 
 	/* source driver precharge control */
 	buf[0] = 0x30;
-	buf[1] = 0x8A;
-	ret = otm8009a_mcs_write(dev, OTM8009A_MCS_SD_PCH_CTRL, buf, 2);
+	ret = otm8009a_mcs_write(dev, OTM8009A_MCS_SD_PCH_CTRL, buf, 1);
```

Comment:
Impact on comment ?

Reply:
`0x8A` is below `OTM8009A_MCS_NO_DOC1` register. I think it is incorrectly added to here.

Reply:
My request was to update the comment as it will now be wrong.

Reply:
no impact on the comment